server/controllers/property.controllers.js

```diff
@@ -263,6 +248,16 @@ const PropertyController = {
       })
       .catch((error) => next(error));
   },
+
+  getPortfolio(req, res, next) {
```

Comment:
Lets make this `getOnePortfolio` so that it can be more descriptive. Same thing with the service name below on line 255,

```suggestion
  getOnePortfolio(req, res, next) {
```